Test/RemotePcapTests.cs

```diff
@@ -29,6 +28,7 @@ public class RemotePcapTests
         };
 
         [Test]
+        [Platform(Exclude="MacOsX")]
```

Comment:
`PcapInterfaceNullAuthTest` should be working fine on macos, since we are compiling libpcap with rpcap support

Reply:
oh that makes sense. I'm using brew here and it doesn't install rpcap. Let me see if it works here.